repos/tap/theme/tapColors.js

```diff
@@ -41,5 +41,6 @@ export const tapColors = {
   inactive: `#a6b0c0`,
   accentBackground: '#f9fafe',
   border: `#e1e7ef`,
+  white: '#ffffff',
```

Comment:
`#FFFFFF` can just be pulled from the `theme` => `theme.colors.palette.white01`
No need to duplicate it

Reply:
updated